components/dashboard/src/index.tsx

```diff
@@ -18,4 +18,11 @@ ReactDOM.render(
         </UserContextProvider>
     </React.StrictMode>,
     document.getElementById('root')
-);
\ No newline at end of file
+);
+
+if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
```

Comment:
this is not running in node

Reply:
It is inlined during the build: https://create-react-app.dev/docs/adding-custom-environment-variables